lncfg/onlynet.go

```diff
@@ -0,0 +1,9 @@
+package lncfg
+
+// Onlynet holds the configuration options for the daemon's network
+// connection restrictions.
+type Onlynet struct {
+	Active bool `long:"active" description:"Activate onlynet restrictions"`
```

Comment:
Nit: this flag doesn't really seem needed? You can determine if it's active based on `Clear` or `Onion` being set.

Reply:
It's not, there can just be something on the Onlynet struct that does that